kloppy/infra/serializers/event/wyscout/deserializer_v3.py

```diff
@@ -49,6 +50,7 @@
 
 from ..deserializer import EventDataDeserializer
 from .deserializer_v2 import WyscoutInputs
+from . import wyscout_tags
```

Comment:
I believe that this import is not used anymore.

Reply:
Done.